pkgs/servers/monitoring/grafana/plugins/victoriametrics-logs-datasource/default.nix

```
@@ -0,0 +1,16 @@
+{
+  grafanaPlugin,
+  lib,
+}:
+
+grafanaPlugin {
+  pname = "victoriametrics-logs-datasource";
+  version = "0.14.3";
+  zipHash = "sha256-g/ntmNyWJ9h/eYpZ0gqiESvVfm2fU6/Ci8R7FHIV7AQ=";
+  meta = with lib; {
+    description = "Grafana datasource for VictoriaLogs";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ samw ];
+    platforms = platforms.unix;
+  };
```

```suggestion
  meta = with lib; {
    description = "Grafana datasource for VictoriaLogs";
    license = licenses.asl20;
    maintainers = with maintainers; [ samw ];
    platforms = platforms.unix;
  };
```
```suggestion
  meta = {
    description = "Grafana datasource for VictoriaLogs";
    license = lib.licenses.asl20;
    maintainers = with lib.maintainers; [ samw ];
    platforms = lib.platforms.unix;
  };
```

we might want to avoid `with lib;` on meta. see https://github.com/NixOS/nixpkgs/issues/371862